specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/stable/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,972 @@
+          "description": "The quantity of phone numbers in the search. Should be at least 1.",
+          "maximum": 2147483647,
+          "minimum": 1,
+          "type": "integer"
+        }
+      }
+    },
+    "PhoneNumberCost": {
+      "description": "The incurred cost for a single phone number.",
+      "type": "object",
+      "properties": {
+        "amount": {
+          "format": "double",
+          "description": "The cost amount.",
+          "type": "number"
+        },
+        "currencyCode": {
+          "description": "The ISO 4217 currency code for the cost amount, e.g. USD.",
+          "type": "string"
+        },
+        "billingFrequency": {
+          "description": "The frequency with which the cost gets billed.",
+          "enum": [
+            "monthly"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BillingFrequency",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PhoneNumberSearchResult": {
+      "description": "The result of a phone number search operation.",
+      "type": "object",
+      "properties": {
+        "searchId": {
+          "description": "The search id.",
+          "type": "string"
+        },
+        "phoneNumbers": {
+          "description": "The phone numbers that are available. Can be fewer than the desired search quantity.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "phoneNumberType": {
+          "description": "The phone number's type, e.g. Geographic, or TollFree.",
+          "enum": [
+            "geographic",
+            "tollFree"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "assignmentType": {
+          "description": "Phone number's assignment type.",
+          "enum": [
+            "user",
+            "application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "cost": {
+          "$ref": "#/definitions/PhoneNumberCost"
+        },
+        "searchExpiresBy": {
+          "format": "date-time",
+          "description": "The date that this search result expires and phone numbers are no longer on hold. A search result expires in less than 15min, e.g. 2020-11-19T16:31:49.048Z.",
+          "type": "string"
+        }
+      }
+    },
+    "PhoneNumberPurchaseRequest": {
+      "description": "The phone number search purchase request.",
+      "type": "object",
+      "properties": {
+        "searchId": {
+          "description": "The search id.",
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationError": {
```

Comment:
This should be removed because it's a duplicate of the type in common.json, @0rland0Wats0n found that when building with autorest.

Reply:
Done.